package.json

```diff
@@ -4,15 +4,16 @@
   "description": "Build CMS in few lines of code for different data sources",
   "main": "lib/index.js",
   "scripts": {
-    "test": "jest --coverage",
-    "test:watch": "jest --watch  --coverage",
-    "lint": "eslint src test",
-    "clean": "rimraf lib dist",
-    "build:flow": "flow-copy-source -v -i '**/test/**' src lib",
-    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --copy-files",
-    "build:watch": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --copy-files --watch",
+    "test": "lerna exec -- jest --coverage",
+    "test:watch": "lerna exec -- jest --watch --coverage",
+    "lint": "eslint ./packages/*/src/**/*.js",
+    "clean": "lerna exec --parallel -- rimraf lib dist",
+    "build:flow": "lerna exec --parallel -- flow-copy-source -v -i '**/test/**' src lib",
+    "build:commonjs": "lerna exec --parallel -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --copy-files",
+    "build:watch": "lerna exec --parallel -- cross-env BABEL_ENV=commonjs babel src --out-dir lib --copy-files --watch",
     "build": "npm run build:commonjs && npm run build:flow",
     "prepublishOnly": "npm run clean && npm run check:src && npm run build",
+    "release": "npm run clean && npm run build && lerna publish --force-publish=*",
```

Comment:
do not `--force-publish=*`, this will disable lerna checking changes of packages.

Reply:
Removed `--force-publish=*`.